### ISSUE TITLE
tbloader not working

### ISSUE THREAD

**Issue by FarisMuhovic** [codecat/godot-tbloader#37]
```
Can't open dynamic library: C:/Users/faris/Desktop/faris/BPoCK/addons/tbloader/bin/tbloader.windows.x86_64.dll, error: Error 126: The specified module could not be found.
  .
  core/extension/native_extension.cpp:270 - GDExtension dynamic library not found: C:/Users/faris/Desktop/faris/BPoCK/addons/tbloader/bin/tbloader.windows.x86_64.dll
  Failed loading resource: res://addons/tbloader/tbloader.gdextension. Make sure resources have been imported by opening the project in the editor at least once.
```

**Comment by codecat** [comment]
Duplicate of #36

**Comment by Muha0644** [comment]
> Duplicate of #36

Actually it's not.
This issue happens when loading the plugin into godot and has nothing to do with `ucrtbased.dll`.
[i know this because we work together and my friend here didn't explain this well enough]

On linux the addon loads and works well (except issues with some maps crashing sometimes). On his windows 10 PC it refuses to load at all, and looks to be a gdextenstion dll bug. The file is definitely there and spelled correctly, so the `dynamic library not found` error makes no sense.

We both use godot 4 beta 4, but this is definitely a platform-specific bug

**Comment by codecat** [comment]
> Error 126: The specified module could not be found.

This error usually always means that a dependency module could not be loaded (in this case, the debug libraries, since you typically only have these with C++ development tools installed, such as Visual Studio).

You could verify this by attaching a debugger or looking at something like [procmon](https://learn.microsoft.com/en-us/sysinternals/downloads/procmon) to see what file is actually not being found.

**Comment by Muha0644** [comment]
Ok, we're gonna try that tomorrow, but at least to me it seems pretty obvious that the error is referring to `tbloader.windows.x86_64.dll`... 

That might be an unclear error message, I am mostly just talking out my ass here. I think it's more likely you're right.

**Comment by codecat** [comment]
This should be fixed in 0.4.1: https://github.com/codecat/godot-tbloader/releases/tag/v0.4.1
